### PR TITLE
Feature/set up projects

### PR DESCRIPTION
**Pull request by achocron**

### CODE REVIEW

File: SDK/MobileCenter/Microsoft.Azure.Mobile.WinForms/Ingestion/Http/NetworkStateAdapter.cs [hunk 1]
```diff
@@ -0,0 +1,9 @@
+﻿using System;
+using System.Net.NetworkInformation;
+
+namespace Microsoft.Azure.Mobile.Ingestion.Http
+{
+    //public class NetworkStateAdapter : INetworkStateAdapter
```
Comment (guperrot) [inline SDK/MobileCenter/Microsoft.Azure.Mobile.WinForms/Ingestion/Http/NetworkStateAdapter.cs]: what is that?

File: SDK/MobileCenter/Microsoft.Azure.Mobile.WinForms/Utils/ApplicationLifecycleHelper.cs [hunk 2]
```diff
@@ -0,0 +1,9 @@
+﻿using System;
+
+
+namespace Microsoft.Azure.Mobile.Utils
+{    
+    //public class ApplicationLifecycleHelper : IApplicationLifecycleHelper
+    //{
```
Comment (guperrot) [inline SDK/MobileCenter/Microsoft.Azure.Mobile.WinForms/Utils/ApplicationLifecycleHelper.cs]: and that?

File: SDK/MobileCenter/Microsoft.Azure.Mobile.WinForms/Utils/DeviceInformationHelper.cs [hunk 3]
```diff
@@ -0,0 +1,11 @@
+﻿using System;
+
+namespace Microsoft.Azure.Mobile.Utils
+{
+    /// <summary>
+    /// Implements the abstract device information helper class
+    /// </summary>
+    //public class DeviceInformationHelper : AbstractDeviceInformationHelper
```
Comment (guperrot) [inline SDK/MobileCenter/Microsoft.Azure.Mobile.WinForms/Utils/DeviceInformationHelper.cs]: and that?

File: SDK/MobileCenter/Microsoft.Azure.Mobile.Windows.Shared/IMobileCenterService.cs [hunk 4]
```diff
@@ -21,6 +21,8 @@ public interface IMobileCenterService
         /// Method that is called to signal start of service.
         /// </summary>
         /// <param name="channelGroup">The channel group to which the channel should be added</param>
-        void OnChannelGroupReady(IChannelGroup channelGroup);
+        /// <param name="appSecret">The app secret of the current application</param>
+
+        void OnChannelGroupReady(IChannelGroup channelGroup, string appSecret);
```
Comment (guperrot) [inline SDK/MobileCenter/Microsoft.Azure.Mobile.Windows.Shared/IMobileCenterService.cs]: I guess you need to merge with develop when we merge the other pr.

**Comment by MatkovIvan** [comment]
@guperrot Commented classes already implemented in https://github.com/Microsoft/mobile-center-sdk-dotnet/pull/270

**Pull request closed without merging**